src/engine/loopingcontrol.cpp

```diff
@@ -126,6 +126,8 @@ LoopingControl::LoopingControl(QString group,
                                           true, false, false, 4.0);
     m_pCOBeatLoopSize->connectValueChangeRequest(this,
             SLOT(slotBeatLoopSizeChangeRequest(double)), Qt::DirectConnection);
+    m_pCOBeatLoopSizeIndicator = new ControlObject(ConfigKey(group, "beatloop_size_indicator"));
```

Comment:
Do we have a conclusion that this is the right way to go? I am in doubt. 
Did you consider that: https://github.com/mixxxdj/mixxx/pull/1187#issuecomment-292636845 ?